src/Component/QueryEditor.tsx

```diff
@@ -102,7 +102,7 @@ export const QueryEditor: ComponentType<Props> = ({ datasource, onChange, onRunQ
                 language="json"
                 showLineNumbers={true}
                 showMiniMap={payload.length > 100}
-                value={payload}
+                value={isString(payload) ? payload : JSON.stringify(payload,undefined,2)}
```

Comment:
is there any advantage of isString of this?
```suggestion
                value={typeof payload === 'string' ? payload : JSON.stringify(payload,undefined,2)}
```

if not, I'd like to keep coupling with lodash as small as possible.

Reply:
I don't know who is better. It's OK to use typeof directly.